src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -74,6 +101,38 @@ public override DbCommand GetCommand(DbConnection connection)
             return command;
         }
 
+        /// <summary>
+        /// Runs a query using the where clause to determine if duplicates are found (causes issues when deleting).
+        /// If no duplicates are found, the check passes, else it returns false;
+        /// </summary>
+        private bool CheckForDuplicateDeleteRows(WhereClause where, string input, DbConnection connection)
+        {
+            using (DbCommand command = connection.CreateCommand())
+            {
+                command.CommandText = input;
+                command.Parameters.AddRange(where.Parameters.ToArray());
+                using (DbDataReader reader = command.ExecuteReader())
+                {
+                    try
+                    {
+                        while (reader.Read())
+                        {
+                            //If the count of the row is
```

Comment:
looks like the comment go cut off.  Style note: please add space ` ` after the `//`.